src/helpers/promise.js

```diff
@@ -1,4 +1,5 @@
 /* eslint no-proto:0 */
+/* eslint no-use-before-define:0 */
```

Comment:
Can you explain why you needed to turn this rule off?  There's likely a better way to handle it than disabling in this one file.